apps/bridge/src/components/Nav/DesktopNav.tsx

```diff
@@ -119,7 +119,7 @@ function IconLink({
 
 function DesktopNav({ color }: DesktopNavProps) {
   return (
-    <div className="hidden h-full w-fit flex-grow flex-row items-center justify-between lg:flex">
```

Comment:
@dschlabach given that it's the default, no issues with removing flex-row here, right?

Reply:
no issue